docs/ChangeLog/20200530/PR8901.md

```diff
@@ -0,0 +1,4 @@
+## update AUDIO-feature comment in avr-template
```

Comment:
This file (and breaking changes) are really not required given the scope of the current changes.

This file should be removed.

Reply:
okay - dropping the changelog commit